lexer.py

```
@@ -0,0 +1,100 @@
+    def get( self ) -> str:
+        if int( self.position ) >= len( self.input ) - 1:
+            self.position += 1
+            return end
+        else:
+            self.position += 1
+        return self.input[ self.position ]
+    
+    def peek( self ) -> str:
+        if int( self.position ) >= len( self.input ) - 1:
+            return end
+        else: 
```

Ruff (0.11.9)

Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)